### PR TITLE
Tours carousel

### PR DESCRIPTION
**Pull request by ghost**
## Related Tickets
- ticket redmine: [https://edu-redmine.sun-asterisk.vn/issues/76393](https://edu-redmine.sun-asterisk.vn/issues/76393)

## WHAT (optional)
-Create Carousel to show images of tour.

### CODE REVIEW

**Comment by ghost** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,6 +6,9 @@
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
 
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.0/js/bootstrap.min.js"></script>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: ko dùng cdn nhé, em kéo thư viện về local nha

**Comment by ghost** [comment]
ready

**Review by ghost — COMMENTED** [review]
bị conflict mất rùi

**Comment by ghost** [comment]
ready

**Review by huongnt-2545 — APPROVED** [review]

**Pull request merged**